### PR TITLE
Editor: Move editor toggle commands to the editor package.

### PR DESCRIPTION
**Pull request by youknowriad**
Related #52632 

## What?

This PR moves the "toggle sidebar", "toggle inspector" and "toggle pre publish checks" commands from the edit-post package to the editor package to share with the site editor (also removes the duplication.

## Testing Instructions

- Check that the commands above work properly in both post and site editors.

### CODE REVIEW

File: packages/edit-post/src/commands/use-commands.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+ * WordPress dependencies
+ */
+import { useSelect, useDispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import { fullscreen } from '@wordpress/icons';
+import { useCommand } from '@wordpress/commands';
+import { store as preferencesStore } from '@wordpress/preferences';
+import { store as noticesStore } from '@wordpress/notices';
+
+export default function useCommands() {
```
Comment (youknowriad) [inline packages/edit-post/src/commands/use-commands.js:11]: This file existed before, I just moved it and removed the commands that were moved to the editor package. I didn't really touch the remaining command.

Comment (ntsekouras) [reply packages/edit-post/src/commands/use-commands.js:11]: Should we rename to `useEditPostCommands`?

Comment (youknowriad) [reply packages/edit-post/src/commands/use-commands.js:11]: It's already in the edit post package?

File: packages/edit-site/src/hooks/commands/use-edit-mode-commands.js [hunk 2]
```diff
@@ -171,57 +158,6 @@ function useManipulateDocumentCommands() {
...
-		},
-	} );
-
-	commands.push( {
-		name: 'core/open-block-inspector',
-		label: __( 'Toggle block inspector' ),
-		icon: blockDefault,
-		callback: ( { close } ) => {
-			close();
-			if ( activeSidebar === 'edit-site/block' ) {
```
Comment (ntsekouras) [inline packages/edit-site/src/hooks/commands/use-edit-mode-commands.js:211]: This was broken in trunk `edit-site/block-inspector`..

Comment (youknowriad) [reply packages/edit-site/src/hooks/commands/use-edit-mode-commands.js:211]: seems like it indeed.

**Review by ntsekouras — APPROVED** [review]
LGTM, thanks!

**Pull request merged**